### PR TITLE
PKA 23 Rating of rateable item.

### PR DESCRIPTION
**Pull request by shreeramneupane**
Rating can done by sending params `value` of rating and `item_id` of rateable item.
Show rating on item details with key `rating` with integer value.

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -21,4 +22,13 @@ class Item < ApplicationRecord
   def self.find_by_category(category_id)
     includes(:categories).where(categories: {id: category_id})
   end
+
+  # Provides the overall rating value of item
+  #
+  # @return [Integer], Overall rating divided by count, if no any ratings are found then return -1
+  def rating
+    count = self.ratings.count
+    return -1 if count == 0
```
Comment (poudelprakash) [inline app/models/item.rb]: just wondering why are you returning -1

File: app/models/rating.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class Rating < ApplicationRecord
+  UPPER_BOUND = 5
```
Comment (poudelprakash) [inline app/models/rating.rb]: any plan on moving this to common constants?

Comment (shreeramneupane) [inline app/models/rating.rb]: @poudelprakash, it would be better if we define class related constant on same class.

**Comment by shreeramneupane** [comment]
:ok:

**Comment by poudelprakash** [comment]
:+1:

**Pull request merged**